pandas/tests/frame/test_dtypes.py

```diff
@@ -69,6 +69,10 @@ def test_empty_frame_dtypes_ftypes(self):
         norows_df = pd.DataFrame(columns=list("abc"))
         assert_series_equal(norows_df.dtypes, pd.Series(np.object, index=list("abc")))
 
+        cat = pd.CategoricalDtype()
```

Comment:
Could you add tests for other extension dtypes (e.g. datetime with timezones and/or period)